app/assets/stylesheets/cards.css

```diff
@@ -39,6 +40,21 @@
   }
 }
 
+#using-these-materials {
+  margin-top: -2rem;
```

Comment:
I have an instinct that using negative margins is not ideal, and changing the margin on `.title-container` would be preferred, but this works